initrd-progs/0initrd/init

```diff
@@ -1282,7 +1285,7 @@ if [ "$UNIONFS" = 'overlay' -a -f /pup_new/etc/rc.d/BOOTCONFIG ]; then
  . /pup_new/etc/rc.d/BOOTCONFIG
  for ONE_SFS in $EXTRASFSLIST; do
   LOOP=`losetup -f`
-  find_onepupdrv "" "" "$ONE_SFS" "p"
+  find_onepupdrv "" "" "$ONE_SFS" "p" "1"
```

Comment:
`"1"` or` ""` ?

Reply:
1

Reply:
This is the only case where we should check both psubdir and the root